wandb/sdk/wandb_run.py

```diff
@@ -919,6 +919,16 @@ def finish(self, exit_code=None):
     def join(self, exit_code=None):
         self.finish(exit_code=exit_code)
 
+    def _add_panel(self, visualize_key: str, panel_type: str, panel_config: dict):
```

Comment:
why start with _? is this API supposed to almost-never be used by end users?

Reply:
This API is private for now, we have a mid-level API that we've built on top that users should use instead.